src/trainer.py

```diff
@@ -169,6 +175,8 @@ def append_epoch_board(self, epoch_dists, eps=0, losses=[],
         epoch_dists = np.array(epoch_dists)
         if name == "train":
             lr = self.dqn.scheduler.state_dict()["_last_lr"]
+            if isinstance(lr, list):
```

Comment:
In what case can the learning rate be a list here? Shouldn't it always be a float?

Reply:
It returned a list and threw an error when I tried to print it in the next line, if I didn't take the first element of the list. It seems like the _last_lr is a list if you look at the last line of the step function [here](https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#StepLR).

Reply:
Oh yes that makes sense, thanks